app/controllers/deploys_controller.rb

```diff
@@ -91,11 +91,13 @@ def search
     deploys = Deploy
     deploys = Deploy.where(stage: stages) if stages
     deploys = Deploy.where(job: jobs) if jobs
```

Comment:
this is wrong ... if I have jobs and stages it overrides deploys ... should be `deploys = deploys.where` ?